packages/template/src/lib/stack-app/apps/interfaces/client-app.ts

```diff
@@ -76,6 +77,12 @@ export type StackClientApp<HasTokenStore extends boolean = boolean, ProjectId ex
     },
   }
   & AsyncStoreProperty<"project", [], Project, false>
+  & AsyncStoreProperty<
+    "item",
+    [{ itemId: string, userId: string } | { itemId: string, teamId: string } | { itemId: string, customId: string }],
```

Comment:
can we maybe call the "customId" customCustomerId everywhere?

```suggestion
    [{ itemId: string, userId: string } | { itemId: string, teamId: string } | { itemId: string, customCustomerId: string }],
```